cmd/virtiofs-placeholder/main_test.go

```diff
@@ -0,0 +1,67 @@
+package virtiofs_placeholder_test
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var placeholderBinary string
+
+func init() {
+	flag.StringVar(&placeholderBinary, "placeholder-binary", "_out/cmd/virtiofs-placeholder", "path to virtiofs placeholder binary")
+}
+
+var _ = Describe("the virtiofs placeholder binary", func() {
+	var (
+		sock string
+		cmd  *exec.Cmd
+	)
+	BeforeEach(func() {
+		if !strings.Contains(placeholderBinary, "../../") {
+			placeholderBinary = filepath.Join("../../", placeholderBinary)
+		}
```

Comment:
we could also simplify here, the assignment should remain constant

Reply:
sorry, I don't understand

Reply:
If we could avoid the Contains basically if the path is always the same. But  we can leave it, it is a very simple check